spring-cloud-app-broker-acceptance-tests/src/test/java/org/springframework/cloud/appbroker/acceptance/BrokerPropertiesParameterResolver.java

```diff
@@ -18,20 +18,25 @@
 
 import java.lang.reflect.Method;
 import java.util.Optional;
+
 import org.junit.jupiter.api.extension.ExtensionContext;
 import org.junit.jupiter.api.extension.ParameterContext;
 import org.junit.jupiter.api.extension.ParameterResolutionException;
 import org.junit.jupiter.api.extension.ParameterResolver;
 
-class BrokerPropertiesParameterResolver implements ParameterResolver {
+public class BrokerPropertiesParameterResolver implements ParameterResolver {
```

Comment:
Any particular reason for making the visibility public over here?

Reply:
trying to avoid package private